osbuild.spec

```diff
@@ -39,6 +39,15 @@ Requires:       util-linux
 Requires:       python3-%{pypi_name} = %{version}-%{release}
 Requires:       (%{name}-selinux if selinux-policy-%{selinuxtype})
 
+# osbuild 125 added a new "solver" field and osbuild-composer only
+# supports this since 116
+Conflicts: osbuild-composer <= 115
+
+# This version needs to get bumped every time the osbuild-dnf-json
+# version changes in an incompatible way. Packages like osbuild-composer
+# can depend on the exact API version this way
+Provides: osbuild-dnf-json-api = 7
```

Comment:
Let's move this to the `-depsolve-dnf` sub-package (line 145).

Reply:
Thanks! Good catch, silly me! Updated.